cmake/macros/macro_deal_ii_pickup_tests.cmake

```
@@ -84,6 +84,47 @@ macro(set_if_empty _variable)
 endmacro()
 
 
+function(pad_string_right output _str _length)
+  string(LENGTH "${_str}" _strlen)
+  math(EXPR _strlen "${_length} - ${_strlen}")
+
+  if(_strlen GREATER 0)
+    if(${CMAKE_VERSION} VERSION_LESS "3.14")
```

Shouldn't this be:
```suggestion
    if(${CMAKE_VERSION} VERSION_LESS "3.15")
```

I have version `3.14.4` and it complains regarding `REPEAT`.

---

I don't know -- I found a code snippet on the web and adapted it. Feel free to submit a patch -- if it fails for you, then I assume you're right and that the version number in the snippet was wrong.